tests/integration_tests/import_export_tests.py

```diff
@@ -351,13 +345,6 @@ def test_import_2_slices_for_same_table(self):
         self.assert_slice_equals(slc_2, imported_slc_2)
         self.assertEqual(imported_slc_2.datasource.perm, imported_slc_2.perm)
 
-    def test_import_slices_for_non_existent_table(self):
```

Comment:
I'm not sure what the purpose of this test was. If the table doesn't exist the `get_table(...)` precursor method with throw.